axonserver/src/main/java/io/axoniq/axonserver/admin/eventprocessor/api/api.kt

```diff
@@ -99,6 +100,32 @@ interface EventProcessorAdminService {
      * @param authentication info about the authenticated user
      */
     fun move(identifier: EventProcessorId, segment: Int, target: String, authentication: Authentication): Mono<Void>
+
+    /**
+     * Balance the load for the specified event processor among the connected client.
+     *
+     * @param processor            the event processor name
+     * @param tokenStoreIdentifier the token store identifier of the event processor
+     * @param strategy         the strategy to be used to balance the load
+     */
+    fun loadBalance(processor: String, tokenStoreIdentifier: String, strategy: String, authentication: Authentication): Mono<Void>
+
+    /**
+     * Balance the load for the specified event processor among the connected client.
+     *
+     * @param processor            the event processor name
+     * @param tokenStoreIdentifier the token store identifier of the event processor
+     * @param strategy         the strategy to be used to balance the load
+     */
+    fun autoLoadBalance(processor: String, tokenStoreIdentifier: String, strategy: String, authentication: Authentication): Mono<Void>
```

Comment:
operation does not perform load balancing, it configures the load balancing strategy for a processor.